net-libs/tox/files/tox-bootstrapd.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=-/bin/mkdir /var/run/tox-bootstrapd -p
```

Comment:
Why is openrc using `/run` and systemd `/var/run`? I think you want `/run` in both cases, and then `RuntimeDirectory=` is the option you're probably looking for.